exercises/practice/anagram/.meta/tests.toml

```diff
@@ -34,12 +46,41 @@ description = "detects anagrams using case-insensitive possible matches"
 
 [7cc195ad-e3c7-44ee-9fd2-d3c344806a2c]
 description = "does not detect an anagram if the original word is repeated"
+include = false
```

Comment:
@vaeng Should this test be included in the C++ unit test file or not? 
The canonical data says it's superseded and this line of `tests.toml` says it's not included, but the unit test file now includes it.